templates/zerver/help/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,10 @@
+# Only allow admins to create new streams
+
+By enabling this will only allow administrators to create new streams within
```

Comment:
You might want to rephrase this into "If you are a Zulip organization administrator, you can enable a setting that only allows administrators to create new streams in the organization"